k8s/logs-spack-io/deployments.yaml

```diff
@@ -0,0 +1,352 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: logs-spack-io-redirect
+  namespace: logs-spack-io
+  labels:
+    app: logs-spack-io
+    svc: web-redirect
```

Comment:
Probably want to change this to simply `web`.  I've been using `svc: web` for components that directly handle web traffic.